gslib/tests/test_cp.py

```diff
@@ -3807,7 +3807,8 @@ def test_cp_resumable_upload_start_over_http_error(self):
 
   def start_over_error_test_helper(self, http_error_num):
     bucket_uri = self.CreateBucket()
-    fpath = self.CreateTempFile(contents=b'a' * 2 * ONE_KIB)
+    rand_chars = get_random_ascii_chars(size=(ONE_MIB * 4))
```

Comment:
Could you leave a comment here explaining that the size has to be fairly large so that the callback can halt the download before it finishes?

Reply:
Sure thing!